sources/Valkey.Glide/GeoAddOptions.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+namespace Valkey.Glide;
+
+/// <summary>
+/// Options for the GEOADD command.
+/// </summary>
+public class GeoAddOptions
+{
+    /// <summary>
+    /// Gets the conditional change option.
+    /// </summary>
+    public ConditionalChange? ConditionalChange { get; }
+
+    /// <summary>
+    /// Gets whether to return the count of changed elements instead of added elements.
+    /// </summary>
+    public bool Changed { get; }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="GeoAddOptions"/> class with conditional change option.
+    /// </summary>
+    /// <param name="conditionalChange">The conditional change option.</param>
+    public GeoAddOptions(ConditionalChange conditionalChange)
+    {
+        ConditionalChange = conditionalChange;
+        Changed = false;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="GeoAddOptions"/> class with changed option.
+    /// </summary>
+    /// <param name="changed">Whether to return the count of changed elements.</param>
+    public GeoAddOptions(bool changed)
+    {
+        ConditionalChange = null;
+        Changed = changed;
+    }
```

Comment:
Maybe property initializers are a bit cleaner? 🤷 
Not really super familiar with C#...

```suggestion
    /// <summary>
    /// Gets the conditional change option.
    /// </summary>
    public ConditionalChange? ConditionalChange { get; } = null;

    /// <summary>
    /// Gets whether to return the count of changed elements instead of added elements.
    /// </summary>
    public bool Changed { get; } = false;

    /// <summary>
    /// Initializes a new instance of the <see cref="GeoAddOptions"/> class with conditional change option.
    /// </summary>
    /// <param name="conditionalChange">The conditional change option.</param>
    public GeoAddOptions(ConditionalChange conditionalChange)
    {
        ConditionalChange = conditionalChange;
    }

    /// <summary>
    /// Initializes a new instance of the <see cref="GeoAddOptions"/> class with changed option.
    /// </summary>
    /// <param name="changed">Whether to return the count of changed elements.</param>
    public GeoAddOptions(bool changed)
    {
        ConditionalChange = null;
    }
```